tests/integration/aws.py

```diff
@@ -55,6 +55,10 @@ def validate_config(cls, cfg: dict, test_name: str, image: str):
             cfg['instance_type'] = "t3.micro"
         if not 'architecture' in cfg or cfg['architecture'] == "amd64":
             cfg['architecture'] = "x86_64"
+        if not 'boot_mode' in cfg:
+            cfg['boot_mode'] = ""
```

Comment:
We should define `legacy-bios` as default here when `boot_mode` is declared as optional within the config (https://github.com/gardenlinux/gardenlinux/pull/1431/files#diff-dacac2ebf9792f0d23c0f922a744486ded01901957d5281290925acd89cf83acR178).